src/test/cdc/t/002_cdc_create_distributed_table_concurrently.pl

```diff
@@ -0,0 +1,99 @@
+# CDC test for create_distributed_table_concurrently
+use strict;
+use warnings;
+
+use Test::More tests => 3;
```

Comment:
Minor: I think giving the number of the tests is the old way of doing it. Currently you are expected to have `done_testing`, see PG examples: https://github.com/postgres/postgres/commit/549ec201d6132b7c7ee11ee90a4e02119259ba5b

The downside of using an exact number is that backports become painful

Reply:
Ok I can change it to use this method to call "done_testing" at the end. thanks.

Reply:
Ok. I have changed all test cases to use ""done_testing" instead of "tests" count now.